### PR TITLE
NOISSUE: fix envs for nginx

### PR DESCRIPTION
**Pull request by mteodor**
Signed-off-by: Mirko Teodorovic <mirko.teodorovic@gmail.com>

### What does this do?

This fixes setting ports for nginx with `.env` file

### CODE REVIEW

**Review by dborovcanin — APPROVED** [review]

File: docker/nginx/entrypoint.sh [hunk 1]
```diff
@@ -13,6 +13,8 @@ envsubst '
     ${MF_USERS_HTTP_PORT}
     ${MF_THINGS_HTTP_PORT}
     ${MF_HTTP_ADAPTER_PORT}
+    ${MF_NGINX_MQTT_PORT}
```
Comment (manuio) [inline docker/nginx/entrypoint.sh:16]: Why `MF_NGINX_MQTT_PORT`?

Comment (drasko) [reply docker/nginx/entrypoint.sh:16]: @manuio what would you suggest?

Comment (manuio) [reply docker/nginx/entrypoint.sh:16]: @drasko something without `NGINX`. Maybe `MF_MQTT_BROKER_PORT`

Comment (mteodor) [reply docker/nginx/entrypoint.sh:16]: ok, whatever you find suitable, but this is MQTT port on NGINX, and there is MQTT port on mqtt-adapter and MQTT port on mqtt broker, so I think it is not bad to denote this in naming

Comment (manuio) [reply docker/nginx/entrypoint.sh:16]: ok sorry, my bad

**Review by manuio — APPROVED** [review]
LGTM

**Pull request merged**